vim/plugins.vim

```diff
@@ -29,3 +29,7 @@ Plug 'vim-airline/vim-airline-themes'
 Plug 'vim-ruby/vim-ruby'
 Plug 'vim-scripts/ReplaceWithRegister'
 Plug 'vim-scripts/greplace.vim'
+
+" Local Plugins
+
+source ~/.vim/plugin/squirrel.vim
```

Comment:
I think this line isn't required, because Vim automatically sources all files in `~/.vim/plugin/` when it starts up.

Documentation: http://learnvimscriptthehardway.stevelosh.com/chapters/42.html